Cargo.toml

```diff
@@ -20,6 +20,7 @@ std = []
 
 [build-dependencies]
 parse-zoneinfo = { version = "0.3" }
+regex = "1"
```

Comment:
I think it's best if we use regex with no default features, I believe that the IANA TZDB is always ascii, and total build time will probably go down more with slower regexes than with building the entire regex crate.

I believe that cargo's crate resolution algorithm may (currently) accidentally include all features in build dependencies in the final binary, instead of only true dependencies.

https://docs.rs/regex/1.3.9/regex/index.html#crate-features